### ISSUE TITLE
Audit of our Pepr logs

### ISSUE THREAD

**Issue by cmwylie19** [defenseunicorns/pepr#1360]
### Describe what should be investigated or refactored

As Pepr has grown, we are creating redundant logs, a lot of them don't make sense to display to the user.

Definition of done:
- Does the log message use emoji 
- Does the log message have relevant information 
- log at the right level (see below)
- Try to avoid acronyms or ism that be obvious to a Pepr dev but maybe not a typical user, every single log line should be very obvious and maybe point to logs. 

Use Kubernetes style [guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use) for what level to log things.

**Comment by samayer12** [comment]
Agreed! This is also related to some of the logging work that happened in #1248 and #959

**Comment by samayer12** [comment]
Logging is a textbook example of a [cross-cutting concern](https://en.m.wikipedia.org/wiki/Cross-cutting_concern). If we approach it with a test-driven mindset, we may find that [aspect-oriented programming](https://blog.bitsrc.io/aspect-oriented-programming-in-javascript-c4cb43f6bfcc) could simplify how we perform logging in Pepr. I'd want to go over some smaller demos and discuss pros/cons with the team, first.

**Comment by samayer12** [comment]
We should also double-check the CLI help menu messages that we display for consistency.

**Comment by samayer12** [comment]
For reference, the attached file contains all of the logging that we do in Pepr as of 0.52.2. There are 197 unique log message entries.

[pepr-log-messages.log](https://github.com/user-attachments/files/21514785/pepr-log-messages.log)

Output was generated with these commands:
```shell
grep "console.*(" src/**/*.ts --exclude=\*.test.ts
grep "Log.*(" src/**/*.ts --exclude=\*.test.ts
```

**Comment by cmwylie19** [comment]
> For reference, the attached file contains all of the logging that we do in Pepr as of 0.52.2. There are 197 unique log message entries.
> 
> [pepr-log-messages.log](https://github.com/user-attachments/files/21514785/pepr-log-messages.log)
> 
> Output was generated with these commands:
> 
> grep "console.*(" src/**/*.ts --exclude=\*.test.ts
> grep "Log.*(" src/**/*.ts --exclude=\*.test.ts

This is a very valid add-on issue, as we are discussing this, we will handle this in a second issue and get this fixed but we are not including it in the epic since it is logging framework related
